docs/reference/dapis/chains/chains-list.md

```diff
@@ -9,12 +9,18 @@ require a restart of the dev server-->
 DO NOT place line breaks between HTML element like <div>.
 They will not render as HTML but rather text or you get an error. No idea why!
 -->
-<div class="api3-bc-chains-box" v-for="(chain, index) in chains">
+
+{{Object.keys(chains).length}} chains:
+
+<!-- alias here is the bind key for v-for:
+https://stackoverflow.com/questions/49675988/how-to-get-the-v-for-index-in-vue-js
+-->
+<div class="api3-bc-chains-box" v-for="(chain, alias, index) in chains">
     <div class="api3-bc-chain-token">
-      {{ chain.nativeToken }}
+       {{ chain.nativeToken }} 
     </div>
     <div class="api3-bc-chains-id">
-      Chain Id: <b>{{ chain.id }}</b>
+       ({{index+1}}) Chain Id: <b>{{ chain.id }}</b>
```

Comment:
I'd remove the index from here because it doesn't correspond to anything and is therefore confusing / distracting.

Reply:
Done